doc/release-notes/release-notes-0.11.2.md

```diff
@@ -65,7 +62,13 @@ darkcoind/darkcoin-qt (on Linux).
 0.11.2 changelog
 ----------------
 
-- *to do ..*
+- Masternode Blinding
+- New Masternode Management Class
```

Comment:
Too specific, there was a lot of similar work was done also I beleive. How about more general "Code restructure and modularization" or smth like that?